README.md

```diff
@@ -283,7 +283,9 @@ And finally, you are ready to go! Or at least you should be...
 
 ### Junior Contributors
 
+**Jaume Jurado** | [Contact me][JJS1440] | Phrase bla bla bla...
 
+[JJS1440]: <https://github.com/JJS1440>
```

Comment:
you need to move this line at the after IPerM link. Do this an you be ready to merge.